packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js

```diff
@@ -190,27 +190,26 @@ class LogsInputsAddConfigureCtrl {
           this.input.data,
           this.configuration.logstash,
         )
-          .then(() => this.goToNetworkPage())
+          .then(() => this.goToInputs())
           .finally(() => this.CucControllerHelper.scrollPageToTop()),
     });
     return this.saving.load();
   }
 
-  goToNetworkPage() {
-    this.$state.go('dbaas.logs.detail.inputs.editwizard.networks', {
-      serviceName: this.serviceName,
-      inputId: this.inputId,
-    });
-    return this.$q.resolve();
-  }
-
   getFlowggerLogFormats() {
     return this.LogsInputsService.getFlowggerLogFormats();
   }
 
   getDelimiters() {
     return this.LogsInputsService.getDelimiters();
   }
+
+  goToInputs() {
+    this.$state.go('dbaas.logs.detail.inputs', {
+      serviceName: this.serviceName,
+    });
+    return this.$q.resolve();
```

Comment:
Why using a `$q.resolve` here ? If you want to return a promise, you can directly :
```suggestion
  return this.$state.go
```